keyboards/duck/octagon/keymaps/via/rules.mk

```diff
@@ -0,0 +1,4 @@
+VIA_ENABLE = yes
+LTO_ENABLE = yes
+MOUSEKEY_ENABLE = no
+BOOTMAGIC_ENABLE = lite
```

Comment:
```suggestion
```

Reply:
Thanks for your suggestions. I have updated it on my fork, not sure if anything is needed to update it on the version submitted? let me know if there are anything required from me.